src/profile-logic/transforms.js

```diff
@@ -237,6 +237,11 @@ export function stringifyTransforms(transforms: TransformStack = []): string {
           'Expected to be able to convert a transform into its short key.'
         );
       }
+      // This switch breaks down each transform down into shared groups of what data
+      // they need as defined in src/types/transforms.js. For instance some transforms
+      // need only a funcIndex, while some care about the current implemention, or
+      // other pieces of data. The only reason why they are grouped together is to
+      // not repeat the same line of code.
```

Comment:
Can you please also add a similar comment in parseTransforms ?
Also I believe the last sentence is not useful, but up to you.